src/libopensc/card-openpgp.c

```diff
@@ -44,13 +51,18 @@
 #include <openssl/sha.h>
 #endif /* ENABLE_OPENSSL */
 
+static const char default_cardname[]    = "OpenPGP card";
+static const char default_cardname_v1[] = "OpenPGP card v1.x";
+static const char default_cardname_v2[] = "OpenPGP card v2.x";
+static const char default_cardname_v3[] = "OpenPGP card v3.x";
```

Comment:
Don't use too much information: please remove the `".x"` if you don't have any information about it.

Reply:
Hmmm, I tend to slightly disagree.
In my interpretation "v3" is equivalent "v3.0"
Hence the ".x", to indicate that we not know exactly whether it is "v3.0", "v3.1", ...

If you really insist, I can remove it. But please give it a 2nd thought.

Reply:
ok, whatever